spec/spec_helper_acceptance.rb

```diff
@@ -1,32 +1,24 @@
-require 'beaker-rspec'
+require 'beaker-rspec/spec_helper'
+require 'beaker-rspec/helpers/serverspec'
 
-config = {
-  'main' => {
-    'logdir' => '/var/log/puppet',
-    'vardir' => '/var/lib/puppet',
-    'ssldir' => '/var/lib/puppet/ssl',
-    'rundir' => '/var/run/puppet',
-  },
-}
-
-# Install latest puppet from puppetlabs.com
 hosts.each do |host|
-  on hosts, install_puppet
+  # Install Puppet
+  install_puppet
 end
-# Explicitly configure puppet to avoid warnings
-configure_puppet(config)
 
 RSpec.configure do |c|
-  module_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+  # Project root
+  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
 
+  # Readable test descriptions
   c.formatter = :documentation
 
+  # Configure all nodes in nodeset
   c.before :suite do
-    on 'debian', 'apt-get -y install apt-transport-https'
-
-    # Install this module for testing
-    puppet_module_install(:source => module_root, :module_name => 'telegraf')
-
+    # Install module
+    # We assume the module is in auto load layout
+    puppet_module_install(:source => proj_root, :module_name => 'telegraf')
+    # Install dependancies
     hosts.each do |host|
```

Comment:
We still need the `apt-transport-https` package installed on Debian, otherwise the Telegraf package fails to install as the InfluxData repo is HTTPS.  Re-adding the `on 'debian', 'apt-get -y install apt-transport-https'` line will do the trick.